agent/consul/util.go

```diff
@@ -443,7 +443,7 @@ func ServersGetACLMode(provider checkServersProvider, leaderAddr string, datacen
 
 // InterpolateHIL processes the string as if it were HIL and interpolates only
 // the provided string->string map as possible variables.
-func InterpolateHIL(s string, vars map[string]string) (string, error) {
+func InterpolateHIL(s string, vars map[string]string, forceValuesToLowercase bool) (string, error) {
```

Comment:
Nitpick, could we simplify the variable name of `forceValuesToLowercase` to something like `lower` or `lowercase`, I think `lowercase = true` makes sense but maybe this isn't friendly?

Reply:
I guess semantically the things we're interpolating _are_ the values, so having the values be in the variable name is redundant. And at that point `lowercase` makes sense.